content/ADR-173-scene-deployments-must-not-contain-world-configuration-section.md

```diff
@@ -33,21 +29,14 @@ In order to deploy a scene to the Worlds Content Server it is mandatory to add a
 }
 ```
 
-This contains metadata that is important at the time of deployment. It may
-also contain other configuration, like skybox settings, comms transport to
-use, etc. None of these settings have any effect in the Catalyst Network, this
-information is completely ignored and may lead to confusion.
+This contains metadata that is important at the time of deployment. It may also contain other configuration, like skybox settings, communications transport to use, etc. None of these settings have any effect in the Catalyst Network, this information is completely ignored and may lead to confusion.
```

Comment:
```suggestion
This contains metadata that is important at the time of deployment. It may also include other configurations, like skybox settings, communications transport to use, etc. None of these settings affect the Catalyst Network. This information is completely ignored and may lead to confusion.
```